db/db.go

```diff
@@ -65,6 +65,13 @@ func InitDB() {
 	db.AutoMigrate(&LeaderBoard{})
 	db.AutoMigrate(&ConnectionCodes{})
 	db.AutoMigrate(&Bounty{})
+	db.AutoMigrate(&Organization{})
+	db.AutoMigrate(&OrganizationUsers{})
+	db.AutoMigrate(&BountyRoles{})
+	db.AutoMigrate(&UserRoles{})
+	db.AutoMigrate(&BountyBudget{})
+	db.AutoMigrate(&BudgetHistory{})
+	db.AutoMigrate(&PaymentHistory{})
```

Comment:
not sure how I feel adding 7 tables for one feature not sure if there is a way to condense this down a bit